### DISCUSSION TITLE
How to remove the home page title?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#998] [Ask a question] [answered]
I think the "index" page would be better without the title...
I don't want to remove the titles of the others pages (like blog, wiki, about...).
I just wish to remove the title of the main page...
I made a search here but I did not find this question...
How do I achieve this, please?

**Comment by ghost** [comment]
I was searching on Google and I found out this link:

[https://v1.docusaurus.io/docs/en/1.10.x/doc-markdown](https://v1.docusaurus.io/docs/en/1.10.x/doc-markdown)

It says that there is an option called "hide_title: " , but it does not show to use it... I tried: 

"hide_title: yes" , or, "hide_title: 1"

but I was not successful.

**Reply by markseuffert** [reply]
Good idea, unfortunately this is not currently supported. 

We thought about a setting like `TitleContent: none` or `ShowTitleContent: 0`. As far as I know, there is no consensus among documentation systems as to which setting should be used for something like this. When there is something that several documentation systems implement, then we can also implement this in Datenstrom Yellow. Feedback is welcome.

**Comment by schulle4u** [comment, marked as answer]
There are several title options available for each page: 

* `Title` = general page title
* `TitleContent` = page title shown in content area below the navigation
* `TitleNavigation` = page title shown in navigation
* `TitleHeader` = page title shown in web browser's title bar
* `TitleSlug` = page title used for saving the page URL slug.

Here is an example home page with different title options: 

```
---
Title: Home page
TitleContent: Introduction
TitleNavigation: Home
---
Page content goes here.
```

Now your home page should display a different content title and avoid redundancy. Is this what you are looking for?

**Reply by ghost** [reply]
Hi @schulle4u!

In fact, I would like to remove the title from the home page...

However, I want to thank you for your tip, a great one! I had no idea that I could use these other options!

It can be my "plan B" if I can't get the title removed.

Thank you very much!

**Comment by markseuffert** [comment]
This can be customised with [HTML layout files](https://datenstrom.se/yellow/help/how-to-customise-a-layout). Have a look at the existing layout files and learn to how make your own layout files. This is one of the core features and I think it's worth learning sooner or later. For example  set `Layout: home` in the [page settings](https://datenstrom.se/yellow/help/how-to-change-the-system#page-settings) at the top of your home page and create a new file `system/layouts/home.html`. Here's an example file to copy/paste:

```
<?php $this->yellow->layout("header") ?>
<div class="content">
<div class="main" role="main">
<?php echo $this->yellow->page->getContentHtml() ?>
</div>
</div>
<?php $this->yellow->layout("footer") ?>
```

Now your home page should display no more page title above the page content. Hope this helps

**Reply by ghost** [reply]
Hi @markseuffert !

What a great tip!!

Yes, sure I will try it! It does not seem to be a hard task, with the example that you wrote.

Thank you very much!

**Reply by ghost** [reply]
Hi again, @markseuffert !

I came back just to tell that you were right: the tip that you talked about really worked!

![thankyou](https://github.com/user-attachments/assets/bfd93f15-4309-4c21-a8a8-e88f433c1996)
